expansion/src/main/java/no/unit/nva/expansion/model/IndexDocumentWrapperLinkedData.java

```diff
@@ -47,8 +46,6 @@
 public class IndexDocumentWrapperLinkedData {
 
     private static final Logger logger = LoggerFactory.getLogger(IndexDocumentWrapperLinkedData.class);
-    private static final String CRISTIN_VERSION = "; version=2023-05-26";
-    private static final String MEDIA_TYPE_JSON_LD_V2 = APPLICATION_JSON_LD.toString() + CRISTIN_VERSION;
```

Comment:
It seems like cristin-api serves the same response when sending 
`application/json` and `application/ld+json; version=2023-05-26`